applications/nrf5340_audio/README.rst

```diff
@@ -1139,6 +1144,13 @@ Given the requirements for the Coordinated Set Identification Service (CSIS), ma
 Dependencies
 ************
 
+The application uses the following |NCS| components:
+
+* :ref:`lib_contin_array`
```

Comment:
This is clear, but in my mind should not need documentation. In the Application KConfigs, all the dependencies are listed. They are also automatically included, so the user need not do anything to make it work.

Reply:
Roger. I've added a note about some of the points you mentioned.
I'll keep this section like this, as all applications and samples have them, but will discuss how we can get rid of redundancies with fellow tech writers.

Reply:
https://projecttools.nordicsemi.no/jira/browse/NCSDK-19309